modules/styleguide/assets/js/e-component.js

```diff
@@ -0,0 +1,127 @@
+import * as commands from './commands';
+import Switcher from './controls/switcher';
+
+require( './lib/dialog' );
```

Comment:
extract the code inside into a function and call it in the constructor of something?

Reply:
Done.

Reply:
Can you please verify and resolve?